lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -431,6 +431,8 @@ class SPIRVModuleImpl : public SPIRVModule {
   std::map<unsigned, SPIRVConstant *> LiteralMap;
   std::vector<SPIRVExtInst *> DebugInstVec;
 
+  SPIRVInstruction *
```

Comment:
A comment describing what this method does would be helpful.

Reply:
Done, uploading soon. The function will be moved to another class, which of course doesn't cancel the comment